scss/components/_dashboard.scss

```diff
@@ -33,6 +35,11 @@
     position: relative;
     top: 2px;
   }
+
+  &--white {
```

Comment:
Do we have a requirement to have this White button?

Reply:
If so, lets avoid having color names and follow semantic naming. maybe `--light`?

Reply:
@deb1990 this was requested by @jamienovick . I made a mistake and added it to another ticket.... but you can read it in the comments of RSE-301 😞

Reply:
Oh Okay. I just wanted to understand where the requirement is coming from. Lets still change the name of the class please.